ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -34,16 +86,55 @@
 <script lang="ts">
 import { defineComponent, reactive, toRefs } from '@vue/composition-api'
 import { RouteNames } from '@/enums'
+import { useGetters } from 'vuex-composition-helpers'
+import { useCountriesProvinces, formatAddress } from '@/composables/address/factories'
 
 export default defineComponent({
   name: 'SubmittingPartyReview',
   components: {},
   props: {},
   setup () {
+    const {
+      getMhrRegistrationSubmittingParty,
+      getMhrAttentionReferenceNum
+    } = useGetters<any>([
+      'getMhrRegistrationSubmittingParty',
+      'getMhrAttentionReferenceNum'
+    ])
+
     const localState = reactive({})
 
+    const parseAddress = () => {
+      let address = getMhrRegistrationSubmittingParty.value.address
+      if (Object.values(address).every((val) => { return !val })) {
+        return '(Not Entered)'
+      }
+      address = formatAddress(address)
+      const street = address.street ? address.street + '<br>' : ''
+      const city = address.city ? address.city + ' ' : ''
+      const region = address.region ? address.region + ' &nbsp;' : ''
+      const postalCode = address.postalCode ? address.postalCode + '<br>' : ''
+      const country = address.country ? useCountriesProvinces().getCountryName(address.country) + '<br>' : ''
+      const details = address.deliveryInstructions ? '<br><i>' + address.deliveryInstructions + '</i>' : ''
+      return `${street}${city}${region}${postalCode}${country}${details}`
+    }
+
+    const parsePhoneNumber = () => {
+      const phone = getMhrRegistrationSubmittingParty.value
+      const phoneNum = phone.phoneNumber
+      const ext = phone.phoneExtension ? ' &nbsp;Ext ' + phone.phoneExtension : ''
+      if (!phoneNum && !ext) {
+        return '(Not Entered)'
+      }
+      return `${phoneNum}${ext}`
```

Comment:
There might be an error here if the `phoneNum` or `ext` are not defined (you have an `&&` check).

Reply:
updated, so now the condition only looks at phone number, and whenever it's undefined, the field will show "not entered"

Reply:
Better, but could be simplified even further: `return phoneNum ?` `${phoneNum}${ext}` `: '(Not Entered)'`